test/caffe2/cuda/test_basic_caffe2_cuda.cc

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gtest/gtest.h>
+
+// #include "tc/c2/context_gpu.h"
+#include "tc/core/cuda/cuda_mapping_options.h"
+
+//#include "isl_cli_strategy.h"
+
+using namespace std;
+// using namespace caffe2;
+// using namespace caffe2::detail;
+using namespace tc;
+
+// TEST(Caffe2, Caffe2) {
+//   IterableInputs ins;
+//   IterableOutputs outs;
+//   IterableArgs args;
+//   OperatorDef def = CreateOperatorDef("MatMulOp", "", ins, outs, args);
+//   def.mutable_device_option()->set_device_type(Backend::Device);
+// }
+
+TEST(TC, TF) {
+  auto options = tc::CudaMappingOptions::makeNaiveMappingOptions();
+}
+
+int main(int argc, char** argv) {
+  ::testing::InitGoogleTest(&argc, argv);
+  ::gflags::ParseCommandLineFlags(&argc, &argv, true);
+  ::google::InitGoogleLogging(argv[0]);
+  return RUN_ALL_TESTS();
+}
+
+#if 0
+
+cd /home/ntv/local/nicolasvasilache/TensorComprehensions-BUILD/build/test/caffe2/cuda
```

Comment:
WAT?

Reply:
you misspelled WUT??!
ugh thanks .. :)

Reply:
I meant this one https://www.destroyallsoftware.com/talks/wat